cpg-core/src/test/kotlin/de/fraunhofer/aisec/cpg/graph/FluentTest.kt

```diff
@@ -70,6 +70,7 @@ class FluentTest {
                     }
                 }
             }
+        val tu = result.translationUnits.firstOrNull()
```

Comment:
Is this to test some exception, at a first glance it looks to me that this line does not do anything and `tu` is not used later in the code.

Reply:
Ah this is probably left over from a previous copy/paste